packages/dd-trace/src/appsec/sdk/set_user.js

```diff
@@ -22,6 +24,19 @@ function setUser (tracer, user) {
   }
 
   setUserTags(user, rootSpan)
+  rootSpan.setTag('_dd.appsec.user.collection_mode', 'sdk')
+
+  const persistent = {}
+
+  if (user.id) {
```

Comment:
This can not be falsy, it is already checked in line 15.

Reply:
indeed